sdk/eventhub/event-hubs/src/eventHubConsumerClient.ts

```diff
@@ -35,14 +35,21 @@ const defaultConsumerClientOptions: Required<Pick<
 };
 
 /**
- * The `EventHubConsumerClient` is the main point of interaction for consuming events in Azure Event Hubs service.
+ * The `EventHubConsumerClient` is the main point of interaction for consuming events from an
+ * Event Hub instance.
  *
  * There are multiple ways to create an `EventHubConsumerClient`
  * - Use the connection string from the SAS policy created for your Event Hub instance.
  * - Use the connection string from the SAS policy created for your Event Hub namespace,
  * and the name of the Event Hub instance
- * - Use the fully qualified domain name of your Event Hub namespace like `<yournamespace>.servicebus.windows.net`,
+ * - Use the fully qualified version of the namespace like `<yournamespace>.servicebus.windows.net`,
  * and a credentials object.
+ * 
+ * Optionally, you can also pass the below:
+ * - An options bag to configure the retry policy or proxy settings.
+ * - A checkpoint store used to store checkpoints. These are the positions from where your application
+ * would resume receiving events when restarting. The checkpoint store is also used by the client 
+ * to balance load among multiple instances of your application.
```

Comment:
```suggestion
 * to load balance multiple instances of your application.
```

"load balance" is a term of art so good to have it literally so people understand quickly and can ctrl+f if needed too.

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-js/pull/6736/commits/03027f8c2c6138d2b112a489ef2ea785faa54c76.